.pre-commit-config.yaml

```diff
@@ -12,6 +12,12 @@ repos:
   - id: debug-statements
   - id: end-of-file-fixer
   - id: trailing-whitespace
+  - id: double-quote-string-fixer
+- repo: https://github.com/psf/black
+  rev: 20.8b1
+  hooks:
+  - id: black
+    args: ["-S", "-l", "127"]
```

Comment:
Please add these arguments into the pyproject.toml config, so tools can pick it up autoamtically. Any reason you went with 127 instead of 120?

Reply:
127 is the size of the Github editor.

Reply:
![fair](https://media.tenor.co/images/b3234de4b15ff0dbbb5071167d4a22fd/tenor.gif)